data/migrations/env.py

```diff
@@ -164,6 +185,17 @@ def run_migrations_online():
     finally:
         connection.close()
 
+    for (revision, previous_revision), migration in revision_to_migration.items():
+        write_migration(migration, revision, previous_revision)
+
+
+def write_migration(migration, revision, previous_revision):
```

Comment:
Can we name this slightly differently, given that it writes the DBA operator migration stuff only?

Reply:
Done